core/storage/collection/gae_models.py

```diff
@@ -558,6 +586,11 @@ def get_deletion_policy():
         """
         return base_models.DELETION_POLICY.KEEP_IF_PUBLIC
 
+    @staticmethod
+    def get_export_policy():
+        """Model does not contain user data."""
+        return base_models.EXPORT_POLICY.NOT_APPLICABLE
```

Comment:
It has editor_ids, viewer_ids, etc. -- so, seems like it does?

Reply:
Changed the docstring. Let me know if it needs to be more verbose.

Reply:
It does need to have more explanation, since it clearly does contain fields like contributor_ids etc.

See my comment on ExpSummaryModel for why we don't export this, despite that. This docstring needs to explain that and point to where the actual export is being done.

Reply:
Updated!